### PR TITLE
Unpin axios after v1.2.2 patch release

### PR DESCRIPTION
**Pull request by dcroote**
Reverts #1553 now that the patch release for the broken v1.2.0 is out

### CODE REVIEW

**Comment by dcroote** [comment]
The `1.2.2` release works so this is ready for review & merge

File: package.json [hunk 1]
```diff
@@ -113,7 +113,7 @@
     "@typescript-eslint/eslint-plugin": "^5.41.0",
     "@typescript-eslint/parser": "^5.41.0",
     "@vercel/ncc": "^0.34.0",
-    "axios": "1.1.3",
+    "axios": "^1.1.3",
```
Comment (Siegrift) [inline package.json:116]: Why not update it to latest version?

**Review by Siegrift — COMMENTED** [review]
Why is yarn.lock not changed?

**Comment by dcroote** [comment]
> Why is yarn.lock not changed?

My intention was to only revert #1553 and let Renovate take care of the update, but I went ahead and bumped it in the latest commit 👍

**Review by Siegrift — APPROVED** [review]
> My intention was to only revert https://github.com/api3dao/airnode/pull/1553 and let Renovate take care of the update, but I went ahead and bumped it in the latest commit 👍

I was just surprised, but it's better to update this to a working version explicitly.

👍 LGTM

**Pull request merged**